ci/aws-lambda-ruby-postgres/3.3/get-dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Gets a zip file for the given lambda layer ARN
+
+# Dev SYS AWS account - use production ARN when available
```

Comment:
Use `# FIXME:` to highlight things like this (other common ones are NOTE, TODO)

Reply:
I've added this as a `TODO`, although tbh I'm a bit torn between that and `FIXME` :thinking:

Reply:
In this case I guess `TODO` is possibly more relevant.  Also on reading on general opinion of the differences, anything with a TODO should be described within a Jira issue - doesn't have to be a separate one - and the Jira number put alongside it.